### PR TITLE
Docker elasticsearch extension

### PR DESCRIPTION
**Pull request by henderkes**
first integration with example usage of elasticsearch, opening localhost:9200 for web access and elasticsearch:9200 for usage within docker

### CODE REVIEW

File: pythonfile/first.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
 from pymongo import MongoClient
+import elastic
+import time
+
+time.sleep(5) #wait for initialisation of elasticsearch and mongodb...
```
Comment (Drakulix) [inline pythonfile/first.py:5]: Okay for testing, but this cannot be the solution in the long term.
We will have to wait for the sockets to become available.
Please add another `FIXME` here, so we don't forget that.

**Review by Drakulix — CHANGES_REQUESTED** [review]
Great work.

File: pythonfile/first.py (hunk 1 above)
Comment (henderkes) [reply pythonfile/first.py:5]: there is a FIXME right under that in the next line, or am I missing something? 80 character limit...
edit: to clarify, the initialisation issue will (have to?) be fixed with exception handling

Comment (Drakulix) [reply pythonfile/first.py:5]: > edit: to clarify, the initialisation issue will (have to?) be fixed with exception handling

So thats the same FIXME then. Good to go in that case.

**Review by Drakulix — APPROVED** [review]

**Pull request merged**